services/relay.md

```diff
@@ -0,0 +1,30 @@
+# Relay
+
+    identifier: 0x183fe656
+
+A switching relay.
+
+## Registers
+
+    rw active: bool @ intensity
```

Comment:
should this be value?

Reply:
No. Maybe cal it "conducting" ?

Reply:
"close" ?

Reply:
I think "closed" may be interpreted the other way by people not familiar with electric circuits; conducting should be clear (if they know the word)

Reply:
conductive it is